src/components/Divider/Divider.props.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ReactNode } from 'react'
+
+import { TextOrientation, Type } from './Divider.types'
+
+export type DividerProps = {
+
+  /**
+   * The wrapped title to be shown in the divider.
+   */
+  children?: ReactNode,
+
+  /**
+   * Show the divider as dashed line. Default is false.
+   */
+  isDashed?: boolean,
+
+  /**
+   * The margin-left/right between the title and its closest border, based on the orientation left or right.
+   * If a numeric value of type string is provided without a unit, it is assumed to be in pixels (px) by default.
+   */
+  orientationMargin?: string | number,
+
+  /**
+   * Defines the text orientation on the divider. Either:
+   *
+   * - center(default)
+   * - left
+   * - right
+   */
+  orientation?: TextOrientation,
+
+  /**
+   * Show the text on divider as plain style. Default is true.
```

Comment:
same here. doc is from antd

Reply:
maybe you can choose to have markdown text on top of the divider?

Reply:
Does this remapping make sense if we don't even know the difference between one prop value and the other? It might make sense to force the divider always to plain for example, if we know it will always be used that way.
Anyway, the difference between plain and non-plain lies in the rendering of the text: `plain` sets a non-heading style of the divider text

Reply:
@RiccardoCorona-MiaPlatform 
while I think you are right, it has happened in the past that our reactord components did not remap all antd props and we had to add them because we needed them later. So I think we should ignore the props that are unclear/useless or that go against our internal patterns (e.g. custom modal height or things like that), but always be very careful when determining what we might and might not need in the future

Reply:
let's only keep the prop that are defined in our design system, not all the ones available from antd